ocp4/profiles/coreos-ncp.profile

```diff
@@ -227,49 +220,51 @@ selections:
     #- package_abrt_removed
 
     ### Login
-    #- disable_users_coredumps
-    #- sysctl_kernel_core_pattern
-    #- coredump_disable_storage
-    #- coredump_disable_backtraces
-    #- service_systemd-coredump_disabled
+    - disable_users_coredumps
+    - sysctl_kernel_core_pattern
+    - coredump_disable_storage
+    - coredump_disable_backtraces
+    - service_systemd-coredump_disabled
     #- var_accounts_max_concurrent_login_sessions=10
     #- accounts_max_concurrent_login_sessions
     #- securetty_root_login_console_only
     #- var_password_pam_unix_remember=5
     #- accounts_password_pam_unix_remember
 
     ### SELinux Configuration
-    #- var_selinux_policy_name=targeted
-    #- selinux_policytype
+    - var_selinux_state=enforcing
+    - selinux_state
+    - var_selinux_policy_name=targeted
+    - selinux_policytype
 
     ### Application Whitelisting (RHEL 8)
-    #- package_fapolicyd_installed
-    #- service_fapolicyd_enabled
-    #- configure_fapolicyd_mounts
+    - package_fapolicyd_installed
+    - service_fapolicyd_enabled
+    - configure_fapolicyd_mounts
 
     ### Enable the Hardware RNG Entropy Gatherer Service
-    #- service_rngd_enabled
+    - service_rngd_enabled
```

Comment:
This is correct and needed now, but in future (with newer kernels or in RHEL with backports), I understand that rngd won't be needed anymore,see https://github.com/openshift/machine-config-operator/issues/854 and https://gitlab.cee.redhat.com/coreos/redhat-coreos/merge_requests/685